lib/ansible/modules/cloud/amazon/aws_kms.py

```diff
@@ -198,11 +204,11 @@ def do_grant(kms, keyarn, role_arn, granttypes, mode='grant', dry_run=True, clea
         if len(changes_needed) and not dry_run:
             policy_json_string = json.dumps(policy)
             kms.put_key_policy(KeyId=keyarn, PolicyName='default', Policy=policy_json_string)
-    except:
-        raise Exception("{}: // {}".format("e", policy_json_string))
+    except Exception as e:
+        raise Exception("{}: // {}".format(e, policy_json_string))
```

Comment:
This code is better than what was here before but do note that catching an exception and rethrowing it like this (in Python2) loses the exception information.  We wouldn't know whether the traceback leads into the json.dumps() call or the kms.put_key_policy() call from the traceback, for instance.  Not a blocker since it is better but something to consider in the future.